skbuild/setuptools_wrap.py

```diff
@@ -609,7 +609,13 @@ def setup(*args, **kw):  # noqa: C901
 
     package_prefixes = _collect_package_prefixes(package_dir, packages)
 
-    _classify_installed_files(cmkr.install(), package_data, package_prefixes,
+    # This hook enables custom processing of the cmake manifest
+    cmake_manifest = cmkr.install()
+    process_manifest = kw.get('process_manifest')
```

Comment:
Since all skbuild options starts with `cmake_` (see [here](https://scikit-build.readthedocs.io/en/latest/usage.html#scikit-build-options)),  what about using the more explicit name `cmake_process_manifest_hook` ?

Reply:
Fixed. Now naming the argument `cmake_process_manifest_hook`.